spec/javascript/controllers/accordion_controller_spec.js

```diff
@@ -32,38 +31,37 @@ describe('AccordionController', () => {
             </div>
         </section>
     </div>
-    ` ;
+    `;
 
     const application = Application.start() ;
     application.register('accordion', AccordionController) ;
 
-    describe("when first loaded", () => {
+    describe("when first loaded with 'active' step set to '2'", () => {
```

Comment:
It may be because `location.href` can be a pain to mock in Jest, but it would be nice to have some coverage of `stepFromURI`.

Reply:
Agree, it was because it's a pain in Jest. Perhaps this is something that would be better handle by Cypress?